pkgs/development/interpreters/nelua/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchFromGitHub, luaPackages }:
```

Comment:
`luaPackages` being an alias can get out of date when lua packages are overriden in an overlay. Using `lua.pkgs` instead reduces that risk